www-misc/profile-sync-daemon: bring back version 5.75

AnAverageHuman opened the pull request:
This commit partially reverts 79f41a54cc55c39e0a4ac9fba8115469f01d262e.
Bug: https://bugs.gentoo.org/678658

juippis commented:
By modern Gentoo standards, USE flags shouldn't be used to control installation of small files, and you should just install them unconditionally. So you should at least remove zsh-completion USE flag, and if that systemd USE is only used to control installation of service files, like it seems to be, you should remove that too and install them unconditionally.

AnAverageHuman commented:
Removed the `zsh-completion` USE flag, but the script seems to prioritize systemd over openrc detection, which is problematic. I've fixed the `usex` command to only install one or the other.

mgorny commented on www-misc/profile-sync-daemon/profile-sync-daemon-5.75.ebuild:
```diff
@@ -0,0 +1,35 @@
+}
+
+pkg_postinst() {
+	elog "The cronjob is -x by default. You might want to set it +x if you"
```
Why would you use systemd timer with USE=-systemd? ;-)

mgorny commented:
Please add yourself as proxy-maint.

AnAverageHuman commented:
Done, but I will not be able to test versions newer than 5.75 as that is the last version that does not require systemd.

mgorny commented:
I suppose that's fine as long as you take care of bumping anyway, and at least build-test them (AFAICS you can do `--buildonly` without systemd installed).
